src/Phinx/Db/Adapter/PdoAdapter.php

```diff
@@ -508,12 +508,11 @@ public function createSchemaTable()
 
             $table = new Table($this->getSchemaTableName(), $options, $this);
 
-            if ($this->getConnection()->getAttribute(\PDO::ATTR_DRIVER_NAME) === 'mysql'
-                && version_compare($this->getConnection()->getAttribute(\PDO::ATTR_SERVER_VERSION), '5.6.0', '>=')) {
```

Comment:
No it is not.
This code

```php
$table->addColumn('version', 'biginteger')
  ->addColumn('migration_name', 'string', array('limit' => 100, 'default' => null, 'null' => true))
  ->addColumn('start_time', 'timestamp')
  ->addColumn('end_time', 'timestamp')
  ->save();
```
will generate the following SQL query:
```sql
CREATE TABLE `phinxlog` (
    `version` BIGINT(20) NOT NULL, 
    `migration_name` VARCHAR(100) NULL, 
    `start_time` TIMESTAMP NOT NULL, 
    `end_time` TIMESTAMP NOT NULL, 
    PRIMARY KEY (`version`)
) ENGINE = InnoDB CHARACTER SET utf8 COLLATE utf8_general_ci;
```
If MySQL is configured with `sql_mode=STRICT_TRANS_TABLES,NO_ZERO_DATE,NO_ZERO_IN_DATE`, then the query won't be executed due to the
`Invalid default value for 'end_time'` error, because there is no default value.

It is logical to use this code:
```php
$table->addColumn('version', 'biginteger')
    ->addColumn('migration_name', 'string', array('limit' => 100, 'default' => null, 'null' => true))
    ->addColumn('start_time', 'timestamp', array('default' => 'CURRENT_TIMESTAMP'))
    ->addColumn('end_time', 'timestamp', array('default' => 'CURRENT_TIMESTAMP'))
    ->save();
```
As a result, the following SQL query will be formed:
```sql
CREATE TABLE `phinxlog` (
    `version` BIGINT(20) NOT NULL, 
    `migration_name` VARCHAR(100) NULL, 
    `start_time` TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP, 
    `end_time` TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP, 
    PRIMARY KEY (`version`)
) ENGINE = InnoDB CHARACTER SET utf8 COLLATE utf8_general_ci;
```
But this will only work with MySQL 5.6 or higher.

----
Because phinx sets timestamps automatically when adding entries to the log, I suggest using `NULL` as the default value, because it will work for all MySQL versions.